lib/services/platform-service.ts

```diff
@@ -223,14 +224,20 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		let platformData = this.$platformsData.getPlatformData(platform, projectData);
 		await this.$pluginsService.validate(platformData, projectData);
 
+		const bundle = appFilesUpdaterOptions.bundle;
+
 		await this.ensurePlatformInstalled(platform, platformTemplate, projectData, config);
-		let changesInfo = this.$projectChangesService.checkForChanges(platform, projectData, { bundle: appFilesUpdaterOptions.bundle, release: appFilesUpdaterOptions.release, provision: config.provision });
+		let changesInfo = this.$projectChangesService.checkForChanges(platform, projectData, { bundle, release: appFilesUpdaterOptions.release, provision: config.provision });
 
 		this.$logger.trace("Changes info in prepare platform:", changesInfo);
 
 		if (changesInfo.hasChanges) {
 			await this.cleanProject(platform, appFilesUpdaterOptions, platformData, projectData);
+		}
+		if (changesInfo.hasChanges || bundle) {
 			await this.preparePlatformCore(platform, appFilesUpdaterOptions, projectData, config, changesInfo, filesToSync);
+		}
+		if (changesInfo.hasChanges) {
```

Comment:
So in case `--bundle` is passed, we'll not update the prepareInfo file. Is this expected?